src/components/GameDetail/index.jsx

```diff
@@ -0,0 +1,96 @@
+import { useEffect, useState } from "react";
+import { Link, useParams } from "react-router-dom";
+import axios from "axios";
+import parse from "html-react-parser";
+import Skeleton, { SkeletonTheme } from "react-loading-skeleton";
+
+import "react-loading-skeleton/dist/skeleton.css";
+import "./index.css";
+
+export default function GameDetail() {
+  const { id: gameId } = useParams();
+  const [gameData, setGameData] = useState({});
+  const {
+    name,
+    background_image,
+    description,
+    genres,
+    developers,
+    released,
+    publishers,
+    website,
+  } = gameData;
+
+  const options = {
```

Comment:
@vinayak-gkmit one suggestion here, instead of passing the API key again and again you can use the Axios instance feature to reduce the repetation.  

https://axios-http.com/docs/instance